Model/Menu/ImportProcessor.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Snowdog\Menu\Model\Menu;
+
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\ValidatorException;
+use Snowdog\Menu\Model\ImportFactory;
+use Snowdog\Menu\Model\Menu\ImportProcessor\Menu as MenuImportProcessor;
+use Snowdog\Menu\Model\Menu\ImportProcessor\Node as NodeImportProcessor;
+
+class ImportProcessor
+{
+    /**
+     * @var ImportFactory
+     */
+    private $importFactory;
+
+    /**
+     * @var MenuImportProcessor
+     */
+    private $menuImportProcessor;
+
+    /**
+     * @var NodeImportProcessor
+     */
+    private $nodeImportProcessor;
+
+    public function __construct(
+        ImportFactory $importFactory,
+        MenuImportProcessor $menuImportProcessor,
+        NodeImportProcessor $nodeImportProcessor
+    ) {
+        $this->importFactory = $importFactory;
+        $this->menuImportProcessor = $menuImportProcessor;
+        $this->nodeImportProcessor = $nodeImportProcessor;
+    }
+
+    /**
+     * @return string
+     */
+    public function importCsv()
+    {
+        $data = $this->uploadFileAndGetData();
+        $menu = $this->createMenu($data);
+
+        $this->nodeImportProcessor->createNodes($data[ExportProcessor::NODES_CSV_FIELD], $menu->getId());
+
+        return $menu->getIdentifier();
+    }
+
+    /**
+     * @return \Snowdog\Menu\Api\Data\MenuInterface
```

Comment:
Could you please import this Interface reference so it matches the rest of the code style?

Reply:
We decided not change this one as `MenuInterface` is not used in the class.